lms/djangoapps/teams/static/teams/templates/team-member.underscore

```diff
@@ -0,0 +1,8 @@
+
+<span class="team-member">
+    <a class="member-profile" href="<%= memberProfileUrl %>">
+        <p><%- gettext(username) %></p>
+        <img class="image-url" src="<%= imageUrl %>" alt="<%- gettext(username) %>" />
+    </a>
+    <span class="sr"><%- gettext("Click to visit profile for this user") %></span>
```

Comment:
This screen reader only text is not necessary.  It's not programmatically linked to the control, so it won't be read along with it.  To effectively pull this off, you would need an ID on the span and then refer to the ID using aria-describedby on the profile page link. I think its too much effort for what its worth.